sql/2020/12_Mobile_Web/12_03.sql

```diff
@@ -0,0 +1,29 @@
+#standardSQL
+# 12_03: Most used apps
+SELECT
+  _TABLE_SUFFIX AS client,
```

Comment:
I assume you're not interested in desktop data here, so I've suggested querying the mobile tables directly.

Reply:
Good call